### DISCUSSION TITLE
What's coming in DaisyUI 4?

### DISCUSSION THREAD

**Discussion by daniellionel01** [saadeghi/daisyui#2460] [General]
Since there's a teaser on the homepage, I'm wondering if there is already any information on it? :)

**Comment by deviprsd** [comment]
https://github.com/saadeghi/daisyui/discussions/2173#discussioncomment-7428600. I know it doesn't answer when, but it is like 1~2 weeks away.

**Reply by deviprsd** [reply]
His twitter seems to have examples https://twitter.com/saadeghi
